deploy/dev/5_load_environment.sh

```diff
@@ -3,17 +3,29 @@ set -euxo pipefail
 
 . utils.sh
 
-function main() {
+main() {
   ./teardown_resources.sh
 
   set_namespace "$APP_NAMESPACE_NAME"
 
-  configure_secret
+  if [ "${DEV_HELM}" = "false" ]; then
+    create_k8s_secret
 
-  deploy_env
+    create_k8s_resources
+
+    deploy_env
```

Comment:
what does `deploy_env` do opposing to `deploy_helm_app`? do they both deploy the same app but one uses helm and one doesn't? if so, let's show it in the naming

Reply:
`deploy_env` deploys the init container flow which is different than how we do it for the helm app we test against

Reply:
i will replace deploy_env -> `deploy_init_env`

Reply:
why use `env` in one place and `app` in another? don't they both deploy an app?